admin_guide/managing_networking.adoc

```diff
@@ -872,6 +879,81 @@ is complicated, because nodes have both "external" IP addresses and
 pass through this chain, but other pod-to-node/master traffic may
 bypass it.
 
+[[enabling-fixed-ips-for-external-project-traffic]]
+== Enabling Fixed IPs for External Project Traffic 
```

Comment:
I did a quick Google on Fixed IPs and all of the references came up as `Static IP`. Is that a better term?
We use Static IP in a few places in our docs, but no Fixed IP that I saw in Atom

Reply:
Hmm I do that these as two separate things, but I'm no expert. @danwinship thoughts?

Reply:
We haven't been very consistent. "fixed", "static", "reserved", "private"...